.clang-format

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
I understand this was generated from the Google C++ style guide. Did we have to change anything or did we just use the auto-generated one directly?

Reply:
This used the auto-generated one directly.